### PR TITLE
Add Swift Configuration Files

### PR DESCRIPTION
**Pull request by mabaiocchi**
This commit adds the "swift_defconfig" configuration file to define
the op-build 'swift' system compile target.  It also adds the
hostboot configuration file "swift.config".

Signed-off-by: Mike Baiocchi <mbaiocch@us.ibm.com>

### CODE REVIEW

**Comment by mabaiocchi** [comment]
@dcrowell77 @crgeddes @wghoffa Please review this PR to add the swift configuration files to op-build.  This will ultimately make the swift-config-wrapper obsolete.

**Comment by mabaiocchi** [comment]
@Erich-Hauptli

File: openpower/configs/swift_defconfig [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+BR2_HOSTBOOT_BINARY_SBE_FILENAME="axone_sbe.img.ecc"
+BR2_OPENPOWER_PNOR_FILENAME="swift.pnor"
+BR2_OPENPOWER_PNOR_XML_LAYOUT_FILENAME="axonePnorLayout_64.xml"
+BR2_OPENPOWER_BIOS_XML_FILENAME="SWIFT_bios.xml"
+BR2_OPENPOWER_MRW_XML_FILENAME="SWIFT_hb.mrw.xml"
+BR2_OPENPOWER_SYSTEM_XML_FILENAME="SWIFT_hb.system.xml"
+BR2_OPENPOWER_MACHINE_XML_FILENAME="swift.xml"
+BR2_OPENPOWER_MACHINE_XML_CUSTOM_GIT_VALUE="git@github.com:open-power/swift-xml.git"
+BR2_OPENPOWER_MACHINE_XML_CUSTOM_GIT=y
+BR2_OPENPOWER_MACHINE_XML_GITHUB_PROJECT_VALUE="swift-xml"
```
Comment (wghoffa) [inline openpower/configs/swift_defconfig:16]: You shouldn't need the custom settings by default. I think it was used in the wrapper to point somewhere other than the default. 

Witherspoon used:
BR2_OPENPOWER_MACHINE_XML_GITHUB_PROJECT_VALUE="witherspoon-xml"
BR2_OPENPOWER_MACHINE_XML_VERSION="0f9b3666becbd5a94d31ff39a2f2a81260c961ed"
BR2_OPENPOWER_MACHINE_XML_FILENAME="witherspoon.xml"
BR2_OPENPOWER_SYSTEM_XML_FILENAME="WITHERSPOON_hb.system.xml"
BR2_OPENPOWER_MRW_XML_FILENAME="WITHERSPOON_hb.mrw.xml"
BR2_OPENPOWER_BIOS_XML_FILENAME="WITHERSPOON_bios.xml"

And no use of custom.

Comment (mabaiocchi) [reply openpower/configs/swift_defconfig:16]: Thanks.  I will remove the XML_CUSTOM lines, but keeping the other CUSTOM lines that witherspoon_defconfig also uses.

**Comment by oohal** [comment]
The Travis errors are because the new defconfig isn't in the tool generated format. Can you run `bash openpower/scripts/update-defconfigs.sh` locally and fold any changes it makes into this PR? You can make it run a bit faster by patching that script to only look at the swift config.

I know it's a bit pedantic, but having the defconfigs in the default format lets us spot any changes to the defconfigs introduced when moving buildroot versions.

**Comment by klauskiwi** [comment]
> 
> 
> The Travis errors are because the new defconfig isn't in the tool generated format. Can you run `bash openpower/scripts/update-defconfigs.sh` locally and fold any changes it makes into this PR? You can make it run a bit faster by patching that script to only look at the swift config.
> 
> I know it's a bit pedantic, but having the defconfigs in the default format lets us spot any changes to the defconfigs introduced when moving buildroot versions.

You can probably manually `./op-build swift_defconfig; ./op-build olddefconfig; ./op-build savedefconfig` and then commit the result

**Comment by mabaiocchi** [comment]
Yes, I'll make the changes and update this PR before we merge it.

**Comment by mabaiocchi** [comment]
> Yes, I'll make the changes and update this PR before we merge it.

I ran `bash openpower/scripts/update-defconfigs.sh` and pushed the resulting change to swift_defconfig.

**Comment by mabaiocchi** [comment]
retest this please

**Comment by klauskiwi** [comment]
lgtm

**Comment by mabaiocchi** [comment]
Thank you, @klauskiwi

**Pull request merged**